### ISSUE TITLE
Home page should only show immediate tournaments

### ISSUE THREAD

**Issue by nathandud** [romrell4/bracket-challenge-ios#17]
Past tournaments clutter up the home page. I recommend showing only current or near future tournaments. We might need to rethink the table view in this case and opt for a modified type of page view.

**Comment by romrell4** [comment]
Definitely. This one is already on my list. It also will eventually slow down the web requests once there's years of tournaments. My original idea was to just simply use sections to divide the two, but I'm thinking of having the endpoint only return a certain timeline, and have the rest only be accessible through some sort of "archive"
